frameworks/Rust/actix/Cargo.toml

```diff
@@ -25,7 +25,7 @@ num_cpus = "1.0"
 futures = "0.1"
 http = "0.1"
 actix = "^0.4.3"
-actix-web = "=0.3.1"
+actix-web = "0.3"
```

Comment:
Just curious, but why are you regressing here?

Reply:
it just "any available version 0.3.x", it it will use current version 0.3.3

Reply:
here is link to travis job, https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/338621337#L2183

Reply:
What's the difference between "^0.4" and "0.4" is that still getting you "0.4.*"? Where possible, we'd like to tie down packages, even to their patch version. The reason for this is, when we run the benchmark rounds and tag the commit, we want to be able to replicate the runs at any time. "0.3" could be "0.3.22" in some future replication, and could create radically different results (or even break the tests - sadly non-breaking semantic versioning is a utopia I only dream of).

Reply:
In fact, we should probably be adding the `cargo.lock` for all rust frameworks

Reply:
"^" mean lowest allowed version. i.e I can say "^0.3.2" - any version starting "0.3.2", "0.3" is any "0.3.x" version. to pin specific version use "=0.3.2" - only 0.3.2. 

Cargo.lock file contains specific version, it solves reproducible builds problem

Reply:
Fantastic, as we just had this issue with a few rust frameworks breaking randomly, let's go with a cargo.lock here.

Reply:
do you want me to commit Cargo.lock for actix in this PR?

Reply:
That would be great @fafhrd91 !

Reply:
added